docs/examples/mht_example.py

```diff
@@ -0,0 +1,248 @@
+# As it is now, there is not a tracker wrapper (as
+# :class:`~.MultiTargetMixtureTracker`) that can be applied directly when dealing with MFA,
+# so we need to specify the tracking loop explicitly.
+
+# load the components
+from stonesoup.predictor.kalman import UnscentedKalmanPredictor
+from stonesoup.updater.kalman import UnscentedKalmanUpdater
+
+predictor = UnscentedKalmanPredictor(transition_model)
+updater = UnscentedKalmanUpdater(measurement_model)
+
+# Data associator and hypothesiser
+from stonesoup.dataassociator.mfa import MFADataAssociator
+from stonesoup.hypothesiser.mfa import MFAHypothesiser
+from stonesoup.hypothesiser.probability import PDAHypothesiser
+
+hypothesiser = PDAHypothesiser(predictor,
+                               updater,
+                               clutter_spatial_density,
+                               prob_gate=0.9999,
+                               prob_detect=prob_detection)
+
+hypothesiser = MFAHypothesiser(hypothesiser)
+data_associator = MFADataAssociator(hypothesiser,
+                                    slide_window=3)
+
+# Prepare the priors
+from stonesoup.types.state import TaggedWeightedGaussianState
+from stonesoup.types.track import Track
+from stonesoup.types.mixture import GaussianMixture
+from stonesoup.types.numeric import Probability
+from stonesoup.types.update import GaussianMixtureUpdate
+
+prior1 = GaussianMixture([TaggedWeightedGaussianState(
+    StateVector([10, 1, 10, 1]),
+    np.diag([10, 1, 10, 1]),
+    timestamp=initial_state.timestamp,
+    weight=Probability(1),
+    tag=[])])
+
+prior2 = GaussianMixture([TaggedWeightedGaussianState(
+    StateVector([-10, -1, -10, -1]),
+    np.diag([10, 1, 10, 1]),
+    timestamp=initial_state.timestamp,
+    weight=Probability(1),
+    tag=[])])
+
+prior3 = GaussianMixture([TaggedWeightedGaussianState(
+    StateVector([-10, -1, 10, 1]),
+    np.diag([10, 1, 10, 1]),
+    timestamp=initial_state.timestamp,
+    weight=Probability(1),
+    tag=[])])
+
+# instantiate the tracks
+tracks = {Track([prior1]),
+          Track([prior2]),
+          Track([prior3])}
+
+# %%
+# 3. Run the tracker and visualise the results;
+# ---------------------------------------------
+# We are ready to loop over the detections in the
+# simulation and obtain the final tracks.
+
+
+for time, detection in tracking:
+    association = data_associator.associate(tracks, detection, time)
+
+    for track, hypotheses in association.items():
+        components = []
+        for hypothesis in hypotheses:
+            if not hypothesis:
+                components.append(hypothesis.prediction)
+            else:
+                update = updater.update(hypothesis)
+                components.append(update)
+        track.append(GaussianMixtureUpdate(components=components,
+                                           hypothesis=hypotheses))
+
+    tracks.add(track)
+
+plotter.plot_tracks(tracks, [0, 2], track_label="Tracks", line=dict(color="Green"))
```

Comment:
Looking at this plot, one track is distracted by the clutter and flies off in a straight line towards the bottom left of the graph. Is this something you want to address, either by commenting on the algorithm's performance or by changing parameters to obtain a better result?